### ISSUE TITLE
CovPass: [a11y] VoiceOver labels missing from graphical controls

### ISSUE THREAD

**Issue by MarcoZehe** [Digitaler-Impfnachweis/covpass-ios#38]
Various elements of the UI are either not accessible (like the QR code image), or have missing labels, like all the buttons that only have graphics as images. Two examples from the main screen are the "Help" and "Plus" buttons, which should get localized strings as AccessibilityLabels such as "Hilfe" and "Hinzufügen" for German. But when I scanned my certificates, I also noticed missing labels in other screens of the app, so a full audit should be performed.

**Comment by jleufgen** [comment]
Hi :) With the next releases the accessibility of the app will be optimized. Thank you for the feedback! Best regards!
